inference-engine/src/mkldnn_plugin/nodes/mkldnn_embedding_bag_packed_sum_node.cpp

```diff
@@ -35,10 +31,8 @@ MKLDNNEmbeddingBagPackedSumNode::MKLDNNEmbeddingBagPackedSumNode(const std::shar
         IE_THROW(NotImplemented) << errorMessage;
     }
 
-    if (op->get_input_shape(INDICES_IDX).size() != 2)
+    if (op->get_input_partial_shape(INDICES_IDX).size() != 2)
```

Comment:
use `getInputShapeAtPort` instead
the same for the other places

Reply:
done, thanks